### ISSUE TITLE
Add text on Accessibility to FreeCEN 'About'

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/FreeCENMigration#921]
Working on the FreeBMD 'About' page I noticed that the accessibility statement (from FreeREG) is not on FreeCEN yet.

**Comment by PatReynolds** [comment]
@AlOneill should I add this text to About-DRAFT on test3?  I would put it as the last item in the page (if no better place is suggested.

**Comment by AlOneill** [comment]
@PatReynolds Your suggestion seems reasonable — it would match the placement on REG.

**Comment by PatReynolds** [comment]
Text added on Test3

**Comment by richpomfret** [comment]
@AlOneill to take a look.

**Comment by AlOneill** [comment]
@PatReynolds Changed the reference to 'FreeREG' to 'FreeCEN'.

The link to the Accessibility Statement is broken — there is no such page.

**Cannot make sync-ready** until either the Accessibility Statement is ready, or we decide to remove the link to that page.

Whilst waiting on the link, I'll make an 'On this page' sub-menu.

**Comment by AlOneill** [comment]
@PatReynolds Have added an 'On this page' sub-menu to the [DRAFT About](https://test3.freecen.org.uk/cms/about-draft) page.

Have also tweaked the source order a little, with phone users in mind.

**Comment by PatReynolds** [comment]
Created draft Accessibility Page (based on FreeREG) on test3 for proofing

**Comment by DeniseColbert** [comment]
The first link is broken (Free and Open Access) - should be https://www.freeukgenealogy.org.uk/news/2016/10/28/access-is-everything

**Comment by PatReynolds** [comment]
Thanks, @DeniseColbert

**Comment by PatReynolds** [comment]
First link now corrected.

**Comment by richpomfret** [comment]
@PatReynolds to deploy.

**Comment by PatReynolds** [comment]
@PatReynolds to deploy on live

**Comment by AlOneill** [comment]
@PatReynolds The page is **not** ready for deployment because the link to the accessibility statement is still broken.

I will investigate.

**Comment by AlOneill** [comment]
@PatReynolds There are two issues here: the Accessibility Statement page is a DRAFT and even if it was not, it is not a child of About, as the `href` states.

Moving back to In Progress and adding a Dependency.

**Comment by PatReynolds** [comment]
This appears to be in part an issue with the instructions for creating new pages: they are supposed to be created outside the structure.  Shall I move the Accessibility statement into the structure?

**Comment by AlOneill** [comment]
The instructions say not to **link** a new page into the structure, which is not the same thing. The main issue is that the Accessibility Statement page is still a DRAFT: that page must be checked and signed off before the About DRAFT can be signed off and made sync-ready.

In the meantime, I've moved the DRAFT Accessibility Statement to be a child of DRAFT About.

**Comment by DeniseColbert** [comment]
@PatReynolds to review Refinery and update story with next step

**Comment by AlOneill** [comment]
@PatReynolds AFAICT, there is just one issue preventing the sign-off of this page: the Accessibility statement is still a draft.
I cannot see a story for the Accessibility statement. The DRAFT page looks pretty good — if there are no urgent changes to be made, I suggest I check it and make it sync-ready. That will allow me to make DRAFT About sync-ready too.

**Comment by AlOneill** [comment]
Now that ARIA attributes are being implemented (and get through to the browser), I've made the DRAFT Accessibility statement live and also the DRAFT About page. The link is no longer broken.

However, there is an "empty" tab spot (visually) between the Accessibility section and the "Top of page" link. @Vino-S why is this so and can we prevent it? Tabbing any further down the page does not work which is a disaster for a keyboard only user.

![About___FreeCEN.png](https://images.zenhubusercontent.com/5ce6b34165b1a70acc0a9124/885372cb-7ef3-44c1-b23f-b1f057b78b5c)

**Comment by DeniseColbert** [comment]
@Vino-S could you take a look please?

**Comment by AlOneill** [comment]
@Vino-S I've become aware of another issue: there is a a version of About — https://test3.freecen.org.uk/cms/about-draft-2 — placed unhelpfully at the foot of the listing and having an advertisement. **It is based on an out of date version of the About page**.

Should the advertisement be moved to the current published About page? Or, more properly, a DRAFT copy of the **current** published page which should then moved to just above About in the Pages listing (to make it obvious that it exists).

**Comment by AlOneill** [comment]
Is there anything more to do for now?

**Comment by DeniseColbert** [comment]
@Vino-S created some dummy pages for My Heritage ad testing so she will take a look at them, remove any surplus ones, and answer your questions @AlOneill

**Comment by AlOneill** [comment]
To investigate 'empty' tab spot and report results to Vino.

**Comment by AlOneill** [comment]
@Vino-S I can now confirm that tabbing is no longer impeded or misleading — mysterious! Closing.

Anyway, I believe this issue is now complete. See #1475 for further work on the About page.
